### DISCUSSION TITLE
3d reconstruction

### DISCUSSION THREAD

**Discussion by proxima22** [Stellarium/stellarium#2887] [Ideas]
Hello
I would like to submit the following idea to you.
I recreated my environment (trees, house, land, car and telescope) by photogrammetry (3d reconstruction / scan of an environment from photo). I would like to know if it would be conceivable to integrate this environment into Stellarium. This would allow me to find the optimal position to observe such a phenomenon at such a time and perhaps the software could even find the ideal position depending on the position of the object against trees and other obstacles on the ground. ?

French 👍 
Bonjour
Je souhaiterais vous soumettre l'idée suivante.
j'ai recréé mon environnement (arbres, maison, terrain, voiture et télescope) par photogrammétrie (reconstitution / scan 3d d'un environnement à partir de photo). Je souhaiterais savoir s'il serait imaginable d'intégrer cet environnement à stéllarium. Cela me permettrait de trouver la position optimale pour observer tel phénomène à tel moment et peut-être même que le logiciel pourrait trouver la position idéale en fonction de la position de l'objet contre vis-à-vis des arbres et autres obstacles au sol ?

**Comment by alex-w** [comment]
Yes, please see Chapter 15 (Scenery3D plugin) in Stellarium User Guide
